### DISCUSSION TITLE
Disabling auto-scheduled PRs, but allowing Dependency Dashboard issued PRs

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#19460] [Request Help] [answered]
I’m looking to have a quiet preset, where only the Dependency Dashboard is updated, but PRs are not raised - without explicit request through the Dashboard - is the solution here to use `schedule: []`, `schedule: ["never"]` or `enabled: false`?

**Comment by rarkins** [comment, marked as answer]
You don't mean dependencyDashboardApproval=true ?

**Reply by jamietanna** [reply]
Yes, that is in fact what I'm looking for, thanks 👏
